docs/reference/troubleshooting/common-issues/task-queue-backlog.asciidoc

```diff
@@ -1,51 +1,90 @@
+<<rejected-requests,rejected requests>>. This may surface restricted to a
+<<data-tiers,data tier>>'s traffic, potentially with <<hotspotting,hot spotting>>
+symptoms.
 
-You can use the <<cat-thread-pool,cat thread pool API>> to 
-see the number of active threads in each thread pool and
-how many tasks are queued, how many have been rejected, and how many have completed. 
+You can use the <<cat-thread-pool,cat thread pool API>> to see the number of
+active threads in each thread pool and how many tasks are queued, how many
+have been rejected, and how many have completed.
 
 [source,console]
 ----
 GET /_cat/thread_pool?v&s=t,n&h=type,name,node_name,active,queue,rejected,completed
 ----
 
+The `active` and `queue` statistics are instantaneous while the `rejected` and
+`completed` statistics are cumulative from node start up.
+
 **Inspect the hot threads on each node**
 
-If a particular thread pool queue is backed up, 
-you can periodically poll the <<cluster-nodes-hot-threads,Nodes hot threads>> API 
-to determine if the thread has sufficient 
-resources to progress and gauge how quickly it is progressing.
+If a particular thread pool queue is backed up, you can periodically poll the
+<<cluster-nodes-hot-threads,Nodes hot threads>> API to determine if the thread
+has sufficient resources to progress and gauge how quickly it is progressing.
 
 [source,console]
 ----
 GET /_nodes/hot_threads
 ----
 
-**Look for long running tasks**
+**Look for long running node tasks**
+
+Long-running tasks can also cause a backlog. You can use the <<tasks,task
+management>> API to get information about the node tasks that are running.
+Check the `running_time_in_nanos` to identify tasks that are taking an
+excessive amount of time to complete.
+
+[source,console]
+----
+GET /_tasks?pretty=true&human=true&detailed=true
+----
+
+If a particular `action` is suspected, you can filter in further. Most common are: 
+
+* <<docs-bulk,bulk index>> related
++
+[source,console]
+----
+GET /_tasks?pretty=true&human=true&detailed=true&actions=indices:data/write/bulk
+----
+
+* search related
++
+[source,console]
+----
+GET /_tasks?pretty=true&human=true&detailed=true&actions=indices:data/write/search
+----
+
+Note the API response may contain tasks columns `description` and `header` 
+which enable futher diagnosising task parameters, target, and requestor. 
 
-Long-running tasks can also cause a backlog. 
-You can use the <<tasks,task management>> API to get information about the tasks that are running. 
-Check the `running_time_in_nanos` to identify tasks that are taking an excessive amount of time to complete. 
+**Look for long running cluster tasks**
+
+Back up may also surface as a delay in synchronizing the cluster state. You
+can use the <<cat-pending-tasks,cat pending tasks API>> to get information
+about the pending cluster state sync tasks that are running. 
 
 [source,console]
 ----
-GET /_tasks?filter_path=nodes.*.tasks
+GET /_cat/pending_tasks?v=true
```

Comment:
This is a totally different API, and I don't think the pending tasks queue has anything to do with task queue backlog. I know the names are confusing in this area.

Reply:
Thanks, Dave! I agree with your call out. 💚

I don't know what's wrong with Github's diff but I did not mix up these APIs. Screenshot of branch from Sublime where I did change the first & then later added the second: 
<img width="540" alt="image" src="https://github.com/user-attachments/assets/03d6c2ae-e44f-41c1-9039-5999a8d46cf4">

Reply:
I see, ok, yeah the Github diff UI isn't showing this very well. Could we point users to the `GET /_cluster/pending_tasks?detailed` API rather than the cat API, aligning with the usage of the `GET /_tasks` API above.